docs/src/inputs/durationPicker/durationPicker.jsx

```diff
@@ -1,351 +1,199 @@
-                        value={this.state.value1}
-                    />
-
-                    <p>
-                        You can use the <code>showXXX</code> props to include or exclude various units of time.<br />
-                        This one has days and hours (by default) and opts in to include minutes and seconds as well.
-                    </p>
-
-                    <DurationPicker
-                        onChange={this._onDurationPicker2Change}
-                        showMinutes
-                        showSeconds
-                        value={this.state.value2}
-                    />
-
-                    <p>This one shows years, months and days, opting out of hours</p>
-
-                    <DurationPicker
-                        onChange={this._onDurationPicker3Change}
-                        showHours={false}
-                        showMonths
-                        showYears
-                        value={this.state.value3}
-                    />
-
-                    <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                        {durationPickerSample}
-                    </Highlighter>
-
-                    {/* Disabled */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
-                        Disabled
-                        <Header.Subheader>
-                            The <code>disabled</code> prop indicates that the Duration Picker is not available for interaction.
-                        </Header.Subheader>
-                    </Header>
-
-                    <DurationPicker
-                        disable
-                        onChange={this._onDuration4PickerChange}
-                        value={this.state.value4}
-                    />
-
-                    <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                        {disabledSample}
-                    </Highlighter>
-
-                    {/* Nest */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
-                        Nest
-                        <Header.Subheader>
-                            A Duration Picker can give the appearance of being nested. The parent's background color needs to be set to <code>color(backgroundColorNest)</code>.
-                        </Header.Subheader>
-                    </Header>
-
-                    <Block
-                        nest
-                        style={{ height: '175px', padding: '22px' }}
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as durationPickerDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/durationPicker/durationPicker';
+/* eslint-enable import/no-named-default, import/extensions */
+
+function DocsDurationPicker() {
+    const descriptionCopy = durationPickerDoc.description;
+
+    return (
+        <Main page="duration_picker">
```

Comment:
Doesn't have to be a blocker, but I just started a pattern to make the creation of these doc pages a little more simple and one the changes is to use the `durationPickerDoc.name` value here as the value for the `page` prop. 

```jsx
const name = durationPickerDoc.name;

return (
    <Main page={camelCase(name)}>
        ...
    </Main>
);

Reply:
Yep, I'll make it so!

Reply:
Fixed in ed7cd01.
For some reason it was `displayName` not `name`.  I wonder if that is maybe a difference between class components and function components.

Reply:
Yes, I saw that accident in my branch as well. Thanks for fixing!